tests/e2e/project/commits.spec.ts

```diff
@@ -87,32 +103,30 @@ test("peer and branch switching", async ({ page }) => {
 });
 
 test("verified badge", async ({ page }) => {
-  if (process.env.HEARTWOOD) {
-    test.skip();
-  }
-
   await page.goto(projectFixtureUrl);
   await page.locator('role=button[name="Commit count"]').click();
 
   await page.getByTitle("Change peer").click();
-  await page.locator("text=bob hyyzz9").click();
-  await expect(page.getByTitle("Change peer")).toHaveText("bob");
-
-  await page.locator("text=Verified").hover();
-
-  await expect(
-    page.locator(
-      "text=This commit was signed with the committer's radicle key.",
-    ),
-  ).toBeVisible();
-  await expect(page.locator(`text=bob committed ${bobRemote}`)).toBeVisible();
-});
-
-test("relative timestamps", async ({ page }) => {
   if (process.env.HEARTWOOD) {
-    test.skip();
+    await page.locator(`text=${bobRemote}`).click();
+    await expect(page.getByTitle("Change peer")).toHaveText(
+      `${bobRemote.substring(0, 6)}…${bobRemote.slice(-6)}`,
+    );
+  } else {
+    await page.locator("text=bob hyyzz9").click();
+    await expect(page.getByTitle("Change peer")).toHaveText("bob");
+    // not applicable to heartwood?
```

Comment:
We don't seem to have a `Verified` badge in heartwood. Thoughts on this @cloudhead ?